docs/generate-time-series/go.rst

```diff
@@ -0,0 +1,374 @@
+
+.. _gen-ts-go:
+
+==================================
+Generate time series data using Go
+==================================
+
+This tutorial will show you how to generate some :ref:`experimental time 
+series data <gen-ts>` from information about the `International Space Station`_
+using `Go`_.
+
+.. SEEALSO::
+
+    :ref:`gen-ts`
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+Prerequisites
+=============
+
+You must have CrateDB :ref:`installed and running <install-run>`.
+
+Make sure you're running an up-to-date version of `Go`_. We recommend 
+Go 1.11 or higher since we will be making use of `modules`_. 
+
+Most of this tutorial is designed to be run as a local Go project using Go 
+tooling since the `compilation`_ unit is the package and not a single line.
+
+To begin, create a project directory and navigate into it:
+
+.. code-block:: console
+
+    $ mkdir time-series-go
+    $ cd time-series-go
+
+Next, choose a module path and create a `go.mod` file that declares it:
```

Comment:
```suggestion
Next, choose a module path and create a ``go.mod`` file that declares it:
```

literals require double backticks

Reply:
(this applies in some other places in this file also)

Reply:
unrelated to the above: what would you think about expanding this a little bit. it's not immediately clear to me what purpose the following command serves